### ISSUE TITLE
Search returns wrong url

### ISSUE THREAD

**Issue by amike80** [rtfd/readthedocs.org#3164]
## Details

* Project URL: https://readthedocs.org/projects/redpitaya/
* Build URL: https://readthedocs.org/projects/redpitaya/builds/6117144/

Search returns the wrong url. However a build on my PC does work 0K.

## Expected Result

Searching for wifi should return the following url:
http://redpitaya.readthedocs.io/en/latest/doc/quickStart/troubleshooting/troubleshooting.html?highlight=wifi

## Actual Result
Search returns wrong url (bellow). However document is build correctly and all tocs and links in the document are working correctly. As can be seen _doc/_ directory is missing in the search url.
http://redpitaya.readthedocs.io/en/latest/quickStart/troubleshooting/troubleshooting.html?highlight=wifi

**Comment by slightlynybbled** [comment]
The first URL that you are referencing actually takes me to the maze.  The 'actual result' URL takes me to a URL that looks appropriate for what you are doing.  In addition, when I actually search for wifi, I get the appropriate links.  As you state, all all actually appears to be OK.

 * When you say 'a build on my PC does work OK', how are you building it on your PC?
 * What is the actual problem?  It looks like all is functioning as desired, simply with a missing `/doc` in the url.  Is this accurate?

**Comment by agjohnson** [comment]
Further, I'd expect the URL path to not include `/doc`, so the hosted version looks exactly as I would expect.

**Comment by amike80** [comment]
Hi,

sorry for the confusion the links for _Expected result_ and _Actual result_ are for the latest document where conf.py was moved inside the the doc folder, this seems to mitigate the issue. We could not leave the document in a state where the search does not work.

Now this is wrong as conf.py is not in the root directory as it should be.

I would've added the links to the specific build (6117144) if I found a way. I also figured that it would be enough to provide a specific build (6117144) url.

For the local builds I've installed sphinx and other required packages on my PC. I tend to build the code locally before pushing it to git.

To build the code i use the following comand:
make -f MakeRTD html
Where MakeRTD is just a renamed version of Make file that comes with sphinx.

Hope this clears up the matter a bit. 

Can anybody proved me with a way to get the urls that show to the specific build and not to the latest build?

@slightlynybbled   It looks like all is functioning as desired, simply with a missing /doc in the url. Is this accurate? That is correct.
